### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by kangawoo44**
- remove textLib usage
- add condition to activity time anchor

### CODE REVIEW

File: resources/base/Receipt.activity.bxb [hunk 1]
```diff
@@ -1,7 +1,13 @@
 activity-support {
   match: Receipt (this)
 
-  time (this.item.dateInterval.start)
+  time {
+    if (exists((this.item.dateInterval.start))) {
+      expression (this.item.dateInterval.start)
+    } else {//by definition dateInterval should have a start & end, so this path is not expected
```
Comment (johncolarusso) [inline resources/base/Receipt.activity.bxb]: i dont think this kind of suggestion is good to have in a sample capsule

Comment (kangawoo44) [reply resources/base/Receipt.activity.bxb]: should we just leave it with a warning then?

Comment (johncolarusso) [reply resources/base/Receipt.activity.bxb]: what if we only have  `expression(now())` and remove if-else?

Comment (kangawoo44) [reply resources/base/Receipt.activity.bxb]: the anchor should really be on `this.item.dateInterval.start`

Comment (johncolarusso) [reply resources/base/Receipt.activity.bxb]: alright. then lets go with your suggestion

**Review by johncolarusso — APPROVED** [review]

File: resources/base/Receipt.activity.bxb (hunk 1 above)
Comment (DogeThis) [reply resources/base/Receipt.activity.bxb]: What does it mean that by definition it has a start and end? there are open intervals. you can have just a start or an end.

Comment (kangawoo44) [reply resources/base/Receipt.activity.bxb]: TIL: "starting next week" type of pattern can lead to only having one of either start or an end even for dateInterval concept.

Comment (kangawoo44) [reply resources/base/Receipt.activity.bxb]: removed comment

**Pull request merged**